jmui/jmui/open_wallet_dialog.ui

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file can be opened via "Qt Designer", which is available on all desktop platforms. On Ubuntu, it can be installed by `sudo apt install qttools5-dev-tools`, and opened by running ` /usr/lib/x86_64-linux-gnu/qt5/bin/designer` after the installation.